packages/apidom-parser/src/parser.ts

```diff
@@ -18,119 +17,124 @@ export type {
 
 export { ParserError };
 
-const ApiDOMParser: stampit.Stamp<ApiDOMParserType> = stampit().init(
-  function ApiDOMParserConstructor() {
-    const adapters: ApiDOMParserAdapter[] = [];
+class ApiDOMParserClass implements ApiDOMParserType {
```

Comment:
```suggestion
class ApiDOMParser {
```
Explanation in https://github.com/swagger-api/apidom/pull/3646/files#r1445972385